nurseconnect/templates/500.html

```diff
@@ -1,21 +1,21 @@
-
 {% load i18n %}
 
 <!DOCTYPE html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-    <head>
-        <meta charset="utf-8" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>{% trans "Internal server error" %}</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-    </head>
-    <body>
-        <h1>{% trans "Internal server error" %}</h1>
+<!--[if gt IE 8]><!-->
+<html class="no-js"> <!--<![endif]-->
```

Comment:
According to the PEP8 rules of my editor, it requires the html tag to be aligned with the closing html tag.